src/components/base/anchor/Anchor.js

```diff
@@ -0,0 +1,282 @@
+    }
+  }, [activeItemId]);
+
+  // Prevent render if not on client
+  if (!mountedOnClient) {
+    return null;
+  }
+
+  // window Y distance from top
+  const scrollY = window.scrollY;
+
+  const windowH = window.innerHeight;
+
+  // Calc height for placeholder (used by the wrap while menu is fixed)
+  const height =
+    (menuWrap.current && menuWrap.current.children[0]?.clientHeight) || 56;
+
+  // Menu distance from top
+  const menuT = (menuWrap.current && menuWrap.current.offsetTop) || windowH;
+
+  // active menu element
+  globalActiveItemId = getActiveElement(items, scrollY, menuT, height);
+
+  // force rerender if the active element has changed
+  if (activeItemId !== globalActiveItemId) {
+    setActiveItemId(globalActiveItemId);
+  }
+
+  // Menu is sticky options
+  const isStickyTop = stickyTop && scrollY > menuT;
+  const isStickyBottom = stickyBottom && windowH < menuT + height - scrollY;
+
+  const isSticky = isStickyTop || isStickyBottom;
+
+  const stickyTopClass = isStickyTop ? styles.stickyTop : "";
+  const stickyBottomClass = isStickyBottom ? styles.stickyBottom : "";
+  const stickyClass = isSticky ? styles.sticky : "";
+
+  // Reuse dynamically added styles from menuItemsWrap rerender
+  const dynamicClassList = [...(itemsWrap.current?.classList || [])];
+  const dynamicStyles = dynamicClassList.filter((c) =>
+    globalDynamicClassList.includes(c)
+  );
+
+  return (
+    <div className={styles.wrap} ref={menuWrap} style={{ height }}>
+      <div
+        className={`${styles.menu} ${dynamicStyles} ${stickyBottomClass} ${stickyTopClass} ${stickyClass}`}
+        ref={itemsWrap}
+      >
+        <Container fluid>
+          <Row>
+            <Col xs={12} lg={{ offset: 3 }}>
+              {Object.keys(items).map((id) => {
+                // target section
+                const section = getSectionById(id, items, menuT, height);
+
+                if (!section) {
+                  return null;
+                }
+
+                // check if element is active
+                const active = activeItemId === id;
+                const activeClass = active ? styles.active : "";
+
+                // Create menu item ref if not already exist
+                itemRefs.current[id] = itemRefs.current[id] ?? createRef();
+
+                // Set item ref
+                const itemRef = itemRefs.current[id];
+
+                return (
+                  <Link
+                    tabIndex="-1"
+                    key={`link-${id}`}
+                    linkRef={itemRef}
+                    className={`anchor-menu-item ${styles.item} ${activeClass}`}
+                    dataCy="anchor-menu-item"
+                    tag="span"
+                    onClick={(e) => {
+                      e.preventDefault();
+                      const target = itemRef.current;
+                      const wrap = itemsWrap.current;
+                      wrap.classList.add(styles.scrolling);
```

Comment:
havde nok foretrukket at undgå ændringer af klasser direkte på dom. Ved at have et state for om den er i gang med at scroll. Man kunne sætte setIsScrolling(true) her, og så som callback til handleScroll, kunne man sætte setIsScrolling(false). Så tror jeg man kunne slippe for de dynamiske klasser